scenario/3_struct_mapping_extend.yml

```diff
@@ -0,0 +1,37 @@
+input:
```

Comment:
Future test cases this feature needs:  (the testcases should fail with an appropriate error message)
- `goverter:mapExtends` with a function reference that doesn't exist
- `goverter:mapExtends` with a non function reference. F.ex `DefaultAge` is a constant. (this should fail)
- `goverter:mapExtends` with a function reference with parameters
- `goverter:mapExtends` with a function reference without return type
- `goverter:mapExtends` with a function reference with wrong return type. Like the output struct requires string but the method return int
- `goverter:mapExtends` with a function reference with multiple return values